package.json

```diff
@@ -21,10 +21,12 @@
     "express-session": "^1.15.6",
     "hashids": "^1.1.4",
     "ip": "^1.1.4",
+    "jsonwebtoken": "^8.1.1",
```

Comment:
Check on npm and make sure its a well used and maintained package.

Reply:
Very well maintained. 6000+ stars and latest commit 3 weeks ago.

Reply:
Great.